docs/reference/installation.rst

```diff
@@ -42,38 +64,42 @@ Next, be sure to enable the bundles in your and ``AppKernel.php`` file:
 
     public function registerbundles()
     {
-        return array(
-            new Sonata\AdminBundle\SonataAdminBundle(),
-            new Sonata\CoreBundle\SonataCoreBundle(),
-            new Sonata\BlockBundle\SonataBlockBundle(),
-            new Sonata\EasyExtendsBundle\SonataEasyExtendsBundle(),
-            // ...
-            new FOS\UserBundle\FOSUserBundle(),
-            new Sonata\UserBundle\SonataUserBundle(),
-            // ...
-        );
+         return array(
```

Comment:
Why did you add an extra space?

Reply:
Didn't see it, will be fixed.

Reply:
and you can use short array syntax here, too